g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/utils/S3Util.java

```diff
@@ -88,18 +87,16 @@ public Locker(String basePath, SingleComponentDTO compDTO) {
 		}
 
 		public boolean lockFile() {
-			long endTime = System.currentTimeMillis() + waitToLock;
 			String content = Double.toString(random.nextDouble());
-
+             int time=0;
```

Comment:
I think time is more intuitionistic than number. We don't know how long it will wait.